pipeline-templates/package-vsix.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
     name: GetVersion
     displayName: '❓ Get Version'
     workingDirectory: $(dir-name)
-  - ${{ if eq('${{ parameters.useOneEngineeringPool }}', 'true') }}:
+  - ${{ if eq(parameters.useOneEngineeringPool, 'true') }}:

```

Comment:
I'm assuming this didn't work correctly before? AFAIK, you can't nest `${{ }}` inside `${{ }}`. It doesn't recursively evaluate like that.

Reply:
that is correct 🆗